docs/details/main-components/animation.rst

```diff
@@ -113,70 +155,193 @@ the following built-in path functions:
+reverse portion of the Animation:
+:cpp:expr:`lv_anim_set_playback_duration(&anim_template, duration_in_ms)`.
+
+Optionally, you can also introduce a delay between the forward and backward
+directions using :cpp:expr:`lv_anim_set_playback_delay(&anim_template, delay_in_ms)`
+
 
-The :cpp:expr:`lv_anim_speed_to_time(speed, start, end)` function calculates the
-required time in milliseconds to reach the end value from a start value
-with the given speed. The speed is interpreted in *unit/sec* dimension.
-For example, :cpp:expr:`lv_anim_speed_to_time(20, 0, 100)` will yield 5000
-milliseconds. For example, in the case of :cpp:func:`lv_obj_set_x` *unit* is
-pixels so *20* means *20 px/sec* speed.
 
-.. _animations_delete:
+.. _animation_start:
+
+Starting an Amination
+*********************
+
+After you have set up your :cpp:type:`lv_anim_t` object, it is important to realize
+that what you have set up is a "template" for a live, running Animation that has
+not been created yet.  When you call :cpp:expr:`lv_anim_start(&anim_template)`
+passing the *template* you have set up, it uses your template to dynamically allocate
+an internal object that is a *live, running* Animation.  This function returns a
+pointer to that object.
+
+.. code-block:: c
+
+    static lv_anim_t   anim_template;
+    static lv_anim_t * running_anim;
+
+    /* Set up template... */
+    lv_anim_init(&anim_template);
+    /* ...and other set-up functions above. */
+
+    /* Later... */
+    running_anim = lv_anim_start(&anim_template);
+
+.. note::
+
+    :cpp:expr:`lv_anim_start(&anim_template)` makes its own copy of the Animation
+    template, so if you do not need it later, its contents do not need to be
+    preserved after this call.
+
+Once a *live running* Animation has been started, it runs until it has completed,
+or until it is deleted (see below), whichever comes first.  An Animation has
+completed when:
+
+- its "value" has reached the desginated ``end`` value;
+- if the Animation has a non-zero *reverse* duration value, then its value
+  has run from the ``end`` value back to the ``start`` value again;
+- if a non-zero repeat count has been set, it has repeated the Animation
+  that number of times.
+
+Once the *live, running* Animation reaches completion, it is automatically deleted
+from the list of running Animations.  This does not impact your Animation template.
+
+.. note::
+
+    If :cpp:expr:`lv_anim_set_repeat_count(&anim_template, cnt)` has been called
+    passing :c:macro:`LV_ANIM_REPEAT_INFINITE`, the animation never reaches a state
+    of being "completed".  In this case, it must be deleted to terminate the
+    Animation.
+
+
+
+.. _animation_delete:
+
+Deleting Animations
+*******************
+
+You should delete an Animation using :cpp:expr:`lv_anim_delete(var, func)` if one of
+these two conditions exists:
+
+- the object (variable) being animated is deleted or
+- a running animation needs to be stopped before it is completed.
```

Comment:
> Could you please mention that animations with an object as their variable are automatically deleted when the object is deleted?

They are???  Wow, I see it now in `lv_obj_destructor()` -- this is hidden because the object doesn't know about the animation and the animation doesn't know about the object, and no `LV_EVENT_DELETE` hook is set by the animation.  BUT!  `lv_anim_delete(widget, NULL)` knows about both....  \*\*rolling eyes at myelf for not having seen this\*\*   Good catch, sir!

Done!